### ISSUE TITLE
Escape key not working

### ISSUE THREAD

**Issue by giuseb** [giuseb/histologyguide#6]
Typing "esc" does not zoom out to the lowest possible magnification on macOS (reproduced on a MacBook Air and a Mac Studio, with both Chrome and Safari)

**Comment by Brelje** [comment]
This was much more difficult than I thought it would be.

From comments in the code, it was disabled when I wanted to use the Escape key to exit my version of full screen (i.e., only hiding the sidebar).  To restore this functionality took some complicated handling of the key press event for the Escape key.

When the sidebar is visible, the Escape key is handled by zoomify and resets to the minimum zoom value.  When the sidebar is hidden (my version of full screen), zoomify skips this keypress so my code can see it and show the sidebar.  In addition, zoomify needs to see it when it is in its full-screen version (i.e., when the space bar is pressed).

One of the quirks of HTML/JavaScript is that the order in which multiple event handlers of the same type (e.g., "keyup") bound to the same element are invoked is not defined.

Easy to describe but harder to implement without introducing "spaghetti" code.  It should work as "expected" in all these cases.

**Comment by giuseb** [comment]
Yes, I can see the complexity! Should I close this issue as resolved?
